api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/argumentAxis/categories.md

```diff
@@ -28,8 +28,13 @@ To sort the arguments, for example, alphabetically, you need to assign an array
         $("#chartContainer").dxChart({
             // ...
             argumentAxis: {
-                categories: ['Africa', 'Antarctica', 'Asia', 'Australia',
-                    'Europe', 'North America', 'South America']
+                categories: 
```

Comment:
The opening square bracket is missing, so are the Americas.

Reply:
Reverted the changes and added the missing square, thanks